src/dotnet/Chat.Service/ChatsBackend.cs

```diff
@@ -675,6 +712,22 @@ private async Task JoinDefaultChatIfAdmin(UserId userId, CancellationToken cance
         await AddOwner(chatId, author, cancellationToken).ConfigureAwait(false);
     }
 
+    private async Task JoinFeedbackTemplateChatIfAdmin(UserId userId, CancellationToken cancellationToken)
+    {
+        var account = await AccountsBackend.Get(userId, cancellationToken).ConfigureAwait(false);
+        if (account is not { IsAdmin: true })
+            return;
+        if (account.Email.IsNullOrEmpty())
+            return;
+        if (!account.Email.OrdinalEndsWith("@actual.chat"))
```

Comment:
I found that we also get email with the following expression `var email = account.User.GetEmail();`
I think this variant is better to use because AccountFull.Email is not verified at the moment.
So everyone can claim that they are from actual.chat domain.
What do you think about?